langchain/agents/tools/requests.ts

```diff
@@ -0,0 +1,69 @@
+import { Tool, Toolkit } from "./base";
+
+type headers = { [key: string]: string };
```

Comment:
Two things here
- types are usually title case so `Headers`
- while this is perfectly valid, types for objects are usually written with the `interface` keyword, like

```
interface Headers {
  [key: string]: string;
}
```